src/common/post.ts

```diff
@@ -274,6 +281,20 @@ export const createSourcePostModeration = async (
     ...args,
     status: SourcePostModerationStatus.Pending,
   });
+
+  const content = `${args.title} ${args.content}`.trim();
```

Comment:
@omBratteng what do you think with this? Since the user can either add a title/content. And title can also be coming from external link generated by our scraper.

Reply:
👍